wdl-lint/src/rules/mixed_indentation.rs

```diff
@@ -126,6 +152,51 @@ impl Visitor for CommandSectionIndentationRule {
 
         // Reset the visitor upon document entry
         *self = Default::default();
+
+        // Store the document text for later use
```

Comment:
this generates and then copies the entire document text without using it later. That is very inefficient. Just use `doc.text().to_string()` directly